pkg/nats/logger.go

```diff
@@ -0,0 +1,44 @@
+package nats
+
+import (
+	"github.com/nats-io/nats-server/v2/server"
+	"github.com/rs/zerolog"
+)
+
+// ZeroLogger is a wrapper around zerolog.Logger to implement the NATS Logger interface
+type ZeroLogger struct {
+	logger zerolog.Logger
+}
+
+// NewZeroLogger creates a new ZeroLogger
+func NewZeroLogger(logger zerolog.Logger) ZeroLogger {
+	return ZeroLogger{
+		logger: logger,
+	}
+}
+
+func (N ZeroLogger) Noticef(format string, v ...interface{}) {
```

Comment:
```suggestion
func (z ZeroLogger) Noticef(format string, v ...interface{}) {
```

Capital for the receiver name seems strange, and might be best to choose a letter from the type name?

Reply:
fixed in https://github.com/bacalhau-project/bacalhau/pull/3188/commits/b88bed4a56822045d404b9d56dbde218bda4040a